sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDBStateStoreProvider.scala

```diff
@@ -352,8 +356,16 @@ private[sql] class RocksDBStateStoreProvider
     var defaultColFamilyId: Option[Short] = None
     if (useColumnFamilies) {
       // put default column family only if useColumnFamilies are enabled
-      colFamilyNameToIdMap.putIfAbsent(StateStore.DEFAULT_COL_FAMILY_NAME, colFamilyId.shortValue())
+      logError(s"### colFamilyIds: $columnFamilyIds")
+      colFamilyNameToIdMap.putAll(columnFamilyIds.asJava)
       defaultColFamilyId = Option(colFamilyId.shortValue())
+      columnFamilyIds.filter(_ != null).map { case (colFamilyName, colFamilyId) =>
```

Comment:
Will default column family be included in `columnFamilyIds` after restart? We'll probably need to check on that

Reply:
Should we have the default column family there?